modules/ROOT/pages/business-groups.adoc

```diff
@@ -0,0 +1,102 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups are also referred to as organizations (or sub organizations). Business groups reside within the master organization and can contain their own business groups. The master organization and its business groups are organized in a hierarchical tree, where the master organization is the root. Each business group has one direct parent and can have multiple children.
```

Comment:
I wouldn't lead with the most confusing thing, that we use two different terms for the same thing and one of those terms also means something else. maybe delete or move the first sentence?

Reply:
I read that whole paragraph and I still don't know what business groups do, what user problem or goal they satisfy.

Reply:
Is the master organization  also a parent business group (the highest biz group?). I think so but am not sure...

Reply:
Also, global comment, it must be either sub-organization or suborganization, not sub organization.